canonical/rebac-admin
# WD-10182 - Update role entitlements

## Done

- Add support for editing role entitlements.

## QA

- Go to the roles page.
- Open the actions menu on a row for a role with existing entitlements and click "Edit".
- Click "Edit entitlements".
- Add and remove some entitlements.
- Go back to the first panel and click "Update role".
- The entitlements should get added and removed.

## Details

- https://warthogs.atlassian.net/browse/WD-10182
